code-challenges.test.js

```diff
@@ -41,13 +64,34 @@ var secretCodeWord3 = "Eccentric"
 var arrayOfWords1 = ["Apple", "Banana", "Plum", "Cherry", "Kiwi"]
 // Expected output: ["Apple", "Banana"]
 var arrayOfWords2 = ["Mango", "Orange", "Apricot", "Blueberry", "Peach"]
-// Expected output: ["Mango", "Banana", "Peach"]
-
+// Expected output: ["Mango", "Apricot", "Peach"]
 
+//make describe and function
+//test for expected  array of words
+describe("includesA", () =>{
+    test ("returns word containg a", () => {
+        expect(includesA(arrayOfWords1)).toEqual(["Apple", "Banana"])
+        expect(includesA(arrayOfWords2)).toEqual(["Mango", "Orange", "Apricot", "Peach"])
 
+    })
+})
 
 
 // b) Create the function that makes the test pass.
+//make function to test
+const includesA = (array) => {
+ //filter through array
+let filtered = array.filter(value => {
+    //checking to see if word has letter
+    let letter = value.includes('a') || value.includes('A')
+  //conditional if word has letter
+        if(letter === true){
```

Comment:
filter has a built in conditional. Same exact answer, less code... great refactor candidate.